addons/contacts/views/contact_views.xml

```diff
@@ -9,9 +9,9 @@
         <field name="context">{'default_is_company': True}</field>
         <field name="help" type="html">
```

Comment:
In your PR message, do not use URLs ( https://tinyurl.com/yfa2ydc7 )
Also the title of the PR can be improved